### PR TITLE
2019 Mid Year Newsletter

### PR DESCRIPTION
**Pull request by jenniferisasi**
This is the markdown file for the first newsletter in a blogpost format and per #1288. Anyone wants to add anything else? It is very short and to the point. I have a trilingual draft to be distributed in PDF format if people like the idea?

### CODE REVIEW

**Comment by mdlincoln** [comment]
Hey @jenniferisasi thank you for drafting this! Is there a reason there needs to be a PDF, and not just this blog post? I don't understand - if the issue is just having different languages, I think it's fine to either put them all on one page, or make three separate blog posts?

**Comment by jenniferisasi** [comment]
> Hey @jenniferisasi thank you for drafting this! Is there a reason there needs to be a PDF, and not just this blog post? I don't understand - if the issue is just having different languages, I think it's fine to either put them all on one page, or make three separate blog posts?

You are right, I can separate the newsletters in three different blogpost. I will work on it today (my internet is limited but hopefully I'll be able to upload them)

**Comment by jenniferisasi** [comment]
Ok. The three files are now uploaded but I need a review by the @programminghistorian/french-team on their version, please.  

I'm going to work on any issue TRAVIS might point to and then change the pull from draft to normal. 
Thank for your patience on my unexpected busy June.

**Review by arojascastro — COMMENTED** [review]
quizás podemos añadir al final que también participaremos en DH2019 con un poster y una charla breve en un workshop sobre pedagogía digital; si necesitas más detalles, te paso. pero en general muy buena idea y muy bien llevada a cabo :)

**Comment by jenniferisasi** [comment]
> quizás podemos añadir al final que también participaremos en DH2019 con un poster y una charla breve en un workshop sobre pedagogía digital; si necesitas más detalles, te paso. pero en general muy buena idea y muy bien llevada a cabo :)

Ay! Pero por supuesto! Estuve dándole vueltas al coco, digo algo me falta de eventos pero de los que me acordé eran eventos potenciales no confirmados. Ahora mismo lo añado. Gracias por estar atento.

File: _posts/2019-06-30-mid-year-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+- Jennifer Isasi
+layout: post
+categories: posts
+---
+
+# Newsletter #1 
+
+Starting in June 2019, and as part of our communication strategies we will  publish a newsletter every three months to summarize our community's progress throughout the year. It is our hope that in this manner we will keep our users informed of lessons and translations, changes in the editorial board, upcoming events organized by our editors and, in all, to keep nurturing a welcoming and multilingual digital humanities community. 
+
+In this first newsletter we would like to highlight the following achievements, which would have been impossible without the help of our great community of editores, reviewers and users.
```
Comment (mdlincoln) [inline _posts/2019-06-30-mid-year-newsletter.md]: `editors` instead of `editores`

File: _posts/2019-06-30-mid-year-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+title: 2019 Mid Year Newsletter 
+authors:
+- Jennifer Isasi
+layout: post
+categories: posts
+---
+
+# Newsletter #1 
+
+Starting in June 2019, and as part of our communication strategies we will  publish a newsletter every three months to summarize our community's progress throughout the year. It is our hope that in this manner we will keep our users informed of lessons and translations, changes in the editorial board, upcoming events organized by our editors and, in all, to keep nurturing a welcoming and multilingual digital humanities community.
```
Comment (mdlincoln) [inline _posts/2019-06-30-mid-year-newsletter.md]: >and as part of our communication strategies

I think you can delete this? Or at least change to 

>Starting in June 2019, as part of our communication strategy, we will publish...

File: _posts/2019-06-30-mid-year-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+## Launch of PH in French
+
+Le [*Programming Historian en français*](http://programminghistorian.org/fr) est en ligne! 
+
+In order to expand our project internationally, *Programming Historian en français* [officially launched](https://programminghistorian.org/posts/welcome-ph-fr) in April 2019. The addition of a third language was only possible thanks to the hard work of the French editorial board and our technical team. Sofia Papastamkou, Marie Puren and François Dominic Laramée worked for more than ten months on the translation of the journal’s infrastructure before its launch. At the same time, the technical team restructured and made the site trilingual.
+
+The team is working on translating existing tutorials. So far, they have published two of our most popular tutorials: François Dominic translated his own lesson [Introduction à la stylométrie en Python](https://programminghistorian.org/fr/lecons/introduction-a-la-stylometrie-avec-python) and Sybille Clochet translated [Nettoyer ses données avec OpenRefine](https://programminghistorian.org/fr/lecons/nettoyer-ses-donnees-avec-openrefine).. The team is currently reviewing four other tutorials.
```
Comment (mdlincoln) [inline _posts/2019-06-30-mid-year-newsletter.md]: Remove double period at end of 

>-donnees-avec-openrefine)..

File: _posts/2019-06-30-mid-year-newsletter.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+The team is working on translating existing tutorials. So far, they have published two of our most popular tutorials: François Dominic translated his own lesson [Introduction à la stylométrie en Python](https://programminghistorian.org/fr/lecons/introduction-a-la-stylometrie-avec-python) and Sybille Clochet translated [Nettoyer ses données avec OpenRefine](https://programminghistorian.org/fr/lecons/nettoyer-ses-donnees-avec-openrefine).. The team is currently reviewing four other tutorials. 
+
+## First Original Lessons in Spanish
+
+Half a year after our successfull Writing Workhop in Bogotá, Colombia, *Programming Historian en español* published its two first original lessons in Spanish. Armando Luza wrote [Creación de corpus paralelo con LF Aligner](https://programminghistorian.org/es/lecciones/corpus-paralelo-lfaligner), a software to help in translation, and Silvia Gutiérrez published [Análisis de corpus con Voyant Tools](https://programminghistorian.org/es/lecciones/analisis-voyant-tools), a guide to the popular text analysis tool.
```
Comment (mdlincoln) [inline _posts/2019-06-30-mid-year-newsletter.md]: Add a link to <https://programminghistorian.org/posts/bogota-workshop-report> ?

File: _posts/2019-06-30-mid-year-newsletter.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+- [Introduction à la stylométrie en Python](https://programminghistorian.org/fr/lecons/introduction-a-la-stylometrie-avec-python) par François Dominic Laramée (traduction)
+
+  Ce tutoriel vous apprendra comment appliquer diverses méthodes stylométriques pour déterminer les auteurs de textes disputés. Trois méthodes sont abordées: les courbes caractéristiques de composition de Mendenhall, la méthode du khi carré de Kilgariff et, enfin, la méthode du Delta de John Burrows.
+
+
+
+## Upcoming Events 
+On July 11, Antonio Rojas Castro, Anna-Maria Sichani and Sofia Papastamkou will present their poster "Designing Multilingual Digital Pedagogy Initiatives: The Programming Historian for English, Spanish, and French speaking DH Communities" at the **DH 2019** Conference in Utrecht, The Netherlands. 
+
+Part of the team of *The Programming Historian en español* will present at the ACH 2019 Conference on July 25, in Pittsburg. Maria José Afanador-Llach, Jennifer Isasi and Antonio Rojas Castro are presenting "Retos en la producción de tutoriales de HD en contextos hispanohablantes" or "Challenges in DH lesson production in Spanish speaking contexts."
```
Comment (mdlincoln) [inline _posts/2019-06-30-mid-year-newsletter.md]: `Pittsburg` should be `Pittsburgh`

Also, maybe add links to the DH 2019 and ACH 2019 conference web pages?

**Review by mdlincoln — CHANGES_REQUESTED** [review]
A few suggestions for the English text.

**Comment by spapastamkou** [comment]
@jenniferisasi added info on DH2019 preconf workshop, in FR and in EN. Could you add it to the ES as well?

File: _posts/2019-06-30-mid-year-newsletter.md (hunk 5 above)
Comment (jenniferisasi) [reply _posts/2019-06-30-mid-year-newsletter.md]: Thank you @mdlincoln for all this comments! I have made the changes and so this should be ready to go (later than I had planned to)

**Comment by jenniferisasi** [comment]
I think we are ready to publish these files! I believe someone needs to approve the final changes to be able to merge, which I cannot do.

**Review by arojascastro — APPROVED** [review]
Amazing work!

**Review by mdlincoln — APPROVED** [review]

**Pull request merged**